ClavaLaraApi/src-lara-clava/clava/clava/graphs/ControlFlowGraph.js

```diff
@@ -22,18 +19,15 @@ class ControlFlowGraph {
   #endNode;
 
   constructor(graph, nodes, startNode, endNode) {
-    this.#graph = graph;
+    super();
```

Comment:
Instead of doing this, I think it would be more robust for the Graph constructor to accept a graph (and the Graph constructor would set the graph)